examples/graph_sage_unsup.py

```diff
@@ -49,10 +33,11 @@ def __init__(self, in_channels, hidden_channels, num_layers):
             in_channels = in_channels if i == 0 else hidden_channels
             self.convs.append(SAGEConv(in_channels, hidden_channels))
 
-    def forward(self, x, adjs):
-        for i, (edge_index, _, size) in enumerate(adjs):
-            x_target = x[:size[1]]  # Target nodes are always placed first.
-            x = self.convs[i]((x, x_target), edge_index)
+    def forward(self, data):
+
+        x, edge_index = data.x, data.edge_index
+        for i in range(self.num_layers):
+            x = self.convs[i]((x, x), edge_index)
```

Comment:
```suggestion
            x = self.convs[i](x, edge_index)
```
I guess this should also work.